### PR TITLE
Form

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**

### CODE REVIEW

**Comment by KAYEMBAJONATHAN** [comment]
- Add form contact 
- Fixt my style 
- Add js file for the functionality
- my coding partner has been not showing and the student success are aware of it

File: index.html [hunk 1]
```diff
@@ -316,20 +316,20 @@ <h2>contact me</h2>
                 </p>
             </div>
             <div class="contact-me-form">
-                <form class="form" action="https://formspree.io/f/mvoyejav" id="form" method="post">
+                <form class="form" action="https://formspree.io/f/mvoyejav" method="post" onsubmit="return validateEmail()">
```
Comment (EricMbouwe) [inline index.html]: - [ ] The form action is always called even if the validation fails. You would move it into your js script in order to trigger the submission only if validation pass. Let's fix it

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN, 

Great work on making the changes requested by a previous reviewer 👏 🔥
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 🎉🔥
- Email validation ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KAYEMBAJONATHAN) [reply index.html]: @EricMbouwe i really to get it your point or undertand

Comment (KAYEMBAJONATHAN) [reply index.html]: i really don't get your point

Comment (KAYEMBAJONATHAN) [reply index.html]: @EricMbouwe

**Comment by ahmetbozaci** [comment]
### Hello @KAYEMBAJONATHAN 
### _This is an additional review as a result of the second opinion request._ 

- According to the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=You%20need%20to%20implement,form%20is%20not%20sent.) if the email is uppercase, the user cannot submit the form. 

- [ ] You can see in the picture that ı typed capitalized email and submit the form successfully without any errors. b Please fix it and submit it for review again.

image





___



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by ishimwezachee — APPROVED** [review]
Hi @KAYEMBAJONATHAN, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights
Email validation ✔
Linters are passing ✔
Project is running successfully ✔ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**